lib/std/build.zig

```diff
@@ -67,6 +67,7 @@ pub const Builder = struct {
     vcpkg_root: VcpkgRoot,
     pkg_config_pkg_list: ?(PkgConfigError![]const PkgConfigPkg) = null,
     args: ?[][]const u8 = null,
+    system_linker_hack: bool,
```

Comment:
No need - we already have this at line 1240. I don't think any changes to this file are needed.